enginetest/enginetests.go

```diff
@@ -5559,9 +5559,14 @@ func findRole(toUser string, roles []*mysql_db.RoleEdge) *mysql_db.RoleEdge {
 func TestBlobs(t *testing.T, h Harness) {
 	h.Setup(setup.MydbData, setup.BlobData, setup.MytableData)
 
+	// By default, strict_mysql_compatibility is disabled, but these tests require it to be enabled.
+	err := sql.SystemVariables.SetGlobal("strict_mysql_compatibility", int8(1))
+	require.NoError(t, err)
 	for _, tt := range queries.BlobErrors {
 		runQueryErrorTest(t, h, tt)
 	}
+	err = sql.SystemVariables.SetGlobal("strict_mysql_compatibility", int8(0))
```

Comment:
setting in a defer might be a bit safer

Reply:
I only wanted `queries.BlobErrors` to do this; I think some of the tests that come later might want the variable to be set to false.